doc/ref_cert/lfn/chapters/chapter02.md

```diff
@@ -14,38 +14,58 @@
 
 <a name="2.1"></a>
 ## 2.1 Introduction
-
-Describe the purpose.. 
-
-- Define E2E Framework Requirements, process and automation for NFVI Certification
-- Ensure Platform Stability, Operability, & Performance for target RAs
+NFVI (Network Functions Virtualization Infrastructure) refers to the physical resources (compute, storage and network) on which virtual network functions (VNFs) are deployed. As such, the performance of a VNF depends on the underlying NFVI over which it is hosted. A certain VNF may perform good in one hardware and may perform worst in another. Thus, a need arises to certify NFVI that can help in onboarding VNFs onto a hardware with an acceptable level of VNF performance. Certain frameworks like [yardstick](https://github.com/opnfv/yardstick), [vsperf](https://github.com/opnfv/vswitchperf) etc. provide a set of tests that can be run on a hardware to obtain Key Performance Indicators (KPIs) which give a measurable output of the NFVI's performance. With these KPIs, a decision can be made on the VNFs that can offer an acceptable level of performance when on-boarded on the NFVI.
 
 <a name="2.2"></a>
 ## 2.2 Methodology
-
-Define the approach to NFVI testing, and Frameworks needed...
+The NFVI provided by hardware vendors is consumed or used by VNFs via APIs exposed by Virtualised Infrastructure Manager (VIM). The resources created by VIM on the NFVI use the underlying physical hardware (compute, storage and network) either directly or indirectly. CNTT recommends RA1/RA2 to be used as a reference architecture for NFVI certification. This  would provide a set of standard interaces to create resources on NFVI. Below step by step process illustrates the NFVI certification methodology:
+* CNTT RA1/RA2 is deployed on NFVI that needs to be certified.
+* A set of tests are run on NFVI to determine the NFVI readiness for certification process,
+* Golden KPIs are taken as a reference.
+* A set of tests are run on the target NFVI.
+* KPIs obtained from the target NFVI are collected and submitted to certification portal.
+* The NFVI KPIs are reviewed and compared with Golden KPIs to determine if the certification badge is to be provided to NFVI or not.
 
```

Comment:
Add additional methodology verbiage:

_Perform VNF interoperability verifications against an implementation of CNTT reference architecture, leveraging existing OPNFV Intake Process. Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.

3rd Party test platforms may also be leveraged, if desired._

![certifying_methodlogy](https://user-images.githubusercontent.com/5643130/71145293-ae60de00-21e6-11ea-9e89-4d3e0282e6a6.jpg)

Figure: Figure: Certification Methodology

Reply:
Added

Reply:
Let me know if we can resolve this.

Reply:
Text looks good.  I could not see the image.  Did you upload the image to the figure directory?  CNTT/doc/ref_cert/lfn/figures/

Reply:
@mf4716 Yes, even I can't see them. All figures are uploaded to ```doc/ref_cert/lfn/figures/``` directory and are showing up in "Files Changed" tab of this PR.

Reply:
Okay.  Let's close/resolve and address any issues after merge.  I'm fairly confident it will work once the images are successfully updated (merged) in the Figures directory.  Resolving this thread.